.circleci/config/jobs/integration_tests_operator_on_k8s.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  docker_layer_caching: true
+  enabled: true
+steps:
+- checkout
+- setup_node12
+- install_python_requests
+- run:
+    command: mkdir -p /tmp/logs/test/integration/kind-olm-operator
+    name: Create temporary directory for logs
+- run:
+    command: |
+      export KUBERNETES_MONITOR_IMAGE_NAME_AND_TAG=$(./scripts/circleci-jobs/setup-integration-tests.py)
+      .circleci/do-exclusively --branch staging --job ${CIRCLE_JOB} npm run test:integration:kindolm:operator
+    name: Operator integration tests on plain k8s
+- run:
+    name: Delete Operator from Quay
+    command: |
+      ./scripts/operator/delete-operator-from-quay.sh
```

Comment:
just making sure there is no race condition here with the regular operator tests (because they both delete the operator)

Reply:
Actually it's one point in the document that Ivan has created about tech debts on operators. For now we think it's ok, because it will only delete the operator in the test.  Because the integration tests take a long time to run and we need it only in the early part of the test to pull the image, it's ok.
However we think we can improve it in the future. Maybe create another pipeline step that will run after all the integration tests (doesn't matter if it has failed or not). 